dashboard/app/controllers/pd/workshop_admins_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Pd::WorkshopAdminsController < ApplicationController
+  authorize_resource class: :pd_workshop_user_management
```

Comment:
This isn't really related to a `pd_workshop_user_management` resource. I don't want to create unnecessary overhead, but I wonder if it might be better to create a "resource" symbol just for this directory, or even just to explicitly check for that permission?

At the very least add a comment stating this resource is a stand in for `workshop_admin` permissions, and if anything changes wrt that resource ability, we'll need to separate them. We do have unit tests, so I'm not very concerned about it sneaking up on us, and it's unlikely to change.

Reply:
I created a dedicated resource symbol (`pd_workshop_admins`) for this page